libsrc/lynx/cgetc.s

```diff
@@ -20,8 +20,11 @@
 ; So the keyboard returns '1', '2', '3', 'P', 'R', 'F' or '?'.
 
 _cgetc:
+        lda     KBEDG
+        bne     _start     
         jsr     _kbhit          ; Check for char available
         beq     _cgetc
```

Comment:
(That `beq` instruction is a part of why the first two lines in this function aren't needed.  And, it's why the `tax` instruction at the end of the `_kbhit` function _is_ needed.)